testing/environments/docker/kafka/Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV KAFKA_HOME /kafka
 # The advertised host is kafka. This means it will not work if container is started locally and connected from localhost to it
 ENV KAFKA_ADVERTISED_HOST kafka
 ENV KAFKA_LOGS_DIR="/kafka-logs"
-ENV KAFKA_VERSION 1.1.1
+ENV KAFKA_VERSION 2.0.0
```

Comment:
thanks for updating this!

Reply:
It would be still pending to test the output with multiple versions, but I leave this for a future change, maybe related with #7957